src/screens/PasswordRecovery/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react'
+import EmailForm from './componentes/EmailForm'
+import * as firebase from 'firebase'
+import { scale, verticalScale } from 'react-native-size-matters/extend'
+import { SvgXml } from 'react-native-svg'
+import { Title, Description } from '~/components/texts'
+import Background from '~/components/Background'
+import styles from './styles'
+import image from '~/assets/images/RecoverPassword/recover.svg'
+import GoBackToSignIn from './componentes/GoBackToSignIn'
+import NavigationService from '~/navigationService'
+
+const PasswordRecovery = () => {
+  const [error, setError] = useState(false)
+
+  const handleSubmit = async (values) => {
+    try {
+      await firebase.auth().sendPasswordResetEmail(values.email)
+      NavigationService.navigate('Login', { passwordRecoveryEmailSent: true })
+    } catch (error) {
+      setError(error)
+    }
+  }
+
+  return (
+    <Background style={styles.container}>
+      <SvgXml width={scale(64)} height={verticalScale(64)} xml={image} marginTop={verticalScale(143)} />
```

Comment:
Move width and height constants into the `./styles` frolder

Reply:
@MPucheta i dont know why but the width and height of a svg does'nt change if it's in style.js, and a looked in other screens to see if there was a motive but i saw that you used it this way.